docs/html/user_guide.rst

```diff
@@ -838,8 +838,8 @@ specifying package versions (e.g. ``~=`` or ``*``):
   semantic versioning.", "``~=3.1``: version ``3.1`` or later, but not version
   ``4.0`` or later. ``~=3.1.2``: version ``3.1.2`` or later, but not
   version ``3.2.0`` or later."
-  ``*``,Can be used at the end of a version number to represent "all", "``== 3.
-  1.*``: any version that starts with ``3.1``. Equivalent to ``~=3.1.0``."
+  ``*``,Can be used at the end of a version number to represent "all", "``== 3.1.*``:
+  any version that starts with ``3.1``. Equivalent to ``~=3.1.0``."
```

Comment:
I believe that this is the result of some editor's auto wrapping to fit the 80-column (IIRC) width.  The following might be slightly better as it fits the constraint, while newlines are employed logically: 
```suggestion
  ``*``, Can be used at the end of a version number to represent "all",
  "``==3.1.*``: any version that starts with ``3.1``.
  Equivalent to ``~=3.1.0``."
```
BTW if you don't feel that is it out of scope, I think it's a good idea to make every cell of the last two columns of this table end with a `.`.